### PR TITLE
Changed the URLs for Pipelines and Service

### PR DESCRIPTION
**Pull request by Kaltner**

### CODE REVIEW

File: src/commands/start.ts [hunk 1]
```diff
@@ -81,15 +81,19 @@ export default class Start extends Command {
     name: string,
     version: string | undefined,
     ops: (OpCommand | OpWorkflow)[],
-  ): boolean => {
-    return ops.some(op => {
+  ): OpCommand | OpWorkflow | undefined => {
+    return ops.find(op => {
```
Comment (maumercado) [inline src/commands/start.ts:85]: wouldn't this break if ops is undefined?

**Review by maumercado — CHANGES_REQUESTED** [review]
Looks really good but just one little question

File: src/commands/start.ts (hunk 1 above)
Comment (Kaltner) [reply src/commands/start.ts:85]: I don't think so, because it is expected to find undefined. However, I will do more tests around it just to be sure

Comment (maumercado) [reply src/commands/start.ts:85]: NVM this looks good, I confused the output with the param

**Review by maumercado — APPROVED** [review]
LGTM

**Pull request merged**